Sources/Likes.php

```diff
@@ -413,7 +413,7 @@ protected function like()
 		}
 
 		// Sometimes there might be other things that need updating after we do this like.
-		call_integration_hook('integrate_issue_like', array($this));
+		call_integration_hook('integrate_issue_like', array($this->_type, $this->_content, $this->_user['id'], $this->_alreadyLiked, $this->_validLikes));
```

Comment:
Its better to pass the entire object and use setter and getters to get access to all properties.